### ISSUE TITLE
[Bug]: Import data but "code":0

### ISSUE THREAD

**Issue by tectal** [milvus-io/milvus#35530]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.4.8
- Deployment mode(standalone or cluster):standalone 
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):2.4.4
- OS(Ubuntu or CentOS): windows
- CPU/Memory: Intel(R) Xeon(R) Silver 4210R CPU @ 2.40GHz，2394 Mhz/128G
- GPU:
- Others:
```

### Current Behavior

C:\Users\jxny02>curl --request POST "http://127.0.0.1:19530/v2/vectordb/jobs/import/create" --header "Content-Type: application/json" --data-raw "{\"files\": [[\"E:/ygf/ygf_code/data/79720492-d1b0-4793-bed0-5f0079c88d60/1.json\"]],\"collectionName\": \"price\"}"

but

{"code":0,"data":{"jobId":"451910421389390804"}}

this my json
[1.json](https://github.com/user-attachments/files/16645157/1.json)

**Comment by xiaofan-luan** [comment]
code 0 actaully means ok I guess. You can use jobID to query whether this job is done.
All import task is async so It will take some time to finish

**Comment by tectal** [comment]
thanks, I use the jobID, but

C:\Users\jxny02>curl --request POST "http://127.0.0.1:19530/v2/vectordb/jobs/import/get_progress" --header "Content-Type: application/json" --data-raw "{\"jobId\": \"451910421390678353\"}"
{"code":0,"data":{"collectionName":"price","completeTime":"","details":[],"fileSize":0,"importedRows":0,"jobId":"451910421390678353","progress":0,"reason":"init failed, failed to decode JSON, error: The specified key does not exist.: importing data failed","state":"Failed","totalRows":0}}

![image](https://github.com/user-attachments/assets/8b1c682b-2645-41a2-96a3-74d9f7e5cfbc)

**Comment by xiaofan-luan** [comment]
init failed, failed to decode JSON, error: The specified key does not exist.: importing data failed","state":"Failed","totalRows":0}


from the error, it is "The specified key does not exist", which means the file you specified is not on S3/Minio

**Comment by yanliang567** [comment]
@tectal you should set the relative path of the milvus bucket on minio as suggested above
![image](https://github.com/user-attachments/assets/eb77a58c-99f3-477e-a9d1-1b8ce5473b6c)
/assign @tectal 
/unassign

**Comment by hscspring** [comment]
> ket on minio
hello, may i ask, what does `Milvus bucket on the MioIO instance` means?

i expose the volume data to the milvus container, and still get `new parquet reader failed, err=parquet: could not retrieve footer offset: The specified key does not exist.: importing data failed`

**Comment by liorf95** [comment]
I am just adding from my side that the import from pre-prepared parquet/JSON files with LocalBulkWriter is not working (getting the "error: The specified key does not exist").
The import works only after the parquet/JSON files are committed by RemoteBulkWriter.

**Comment by hscspring** [comment]
@liorf95 yeah, the `LocalBulkWriter` seems useless in prod environment. we could always use `RemoteBulkWriter`.

**Comment by bigsheeper** [comment]
> @liorf95 yeah, the `LocalBulkWriter` seems useless in prod environment. we could always use `RemoteBulkWriter`.

1. A `LocalBulkWriter` instance rewrites your raw data locally into a format that Milvus understands. It’s useful if you want to preprocess the data before uploading it to Milvus.
2. If you want to directly import your data into Milvus, We recommend using [RemoteBulkWriter](https://milvus.io/api-reference/pymilvus/v2.4.x/DataImport/RemoteBulkWriter/commit.md) instead, which handles data ingestion remotely and can simplify the import process.
